src/main/resources/application.properties

```diff
@@ -15,4 +15,20 @@ spring.jpa.hibernate.ddl-auto = update
 # stripped before adding them to the entity manager)
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL8Dialect
 
+# Email Properties
+# add these new properties
```

Comment:
remove this comment pls

Reply:
thought would be useful for my future reference